examples/pennylane_run_qaoa_maxcut.py

```diff
@@ -0,0 +1,287 @@
+# a given partition of the vertices (blue circles) into two sets as shown in the figure below.
+#
+# .. figure:: ../../examples/figures/qaoa_maxcut_partition.png
+#    :align: center
+#    :scale: 65%
+#    :alt: qaoa_operators
+#
+# |
+# Let's say there are :math:`m` edges. We seek the partition :math:`z` of the vertices into two sets
+# 0 and 1 which maximizes
+#
+# .. math::
+#   C(z) = \sum_{\alpha=1}^{m}C_\alpha(z)
+#
+# where :math:`C_\alpha(z)=1` if :math:`z` places one vertex from the
+# :math:`\alpha^\text{th}` edge in set 0 and the other in set 1, and :math:`C_\alpha(z)=0` otherwise.
+# The goal of approximate optimization in this case is to find a partition :math:`z` which
+# yields a value for :math:`C(z)` that is close to the maximum possible value.
+#
+# For instance,
+# in the situation depicted in the figure above, the optimal value of :math:`C(z)` is
+# 4. Such a situation can be represented by the bitstring :math:`z=1010\text{,}`
+# indicating that the :math:`0^{\text{th}}` and :math:`2^{\text{nd}}` vertices are in one set
+# while the :math:`1^{\text{st}}` and :math:`3^{\text{rd}}` are in
+# the other. (The inverse partition, :math:`z=0101` is of course equally valid.) In the following sections,
+# we will represent partitions using computational basis states and use PennyLane to
+# rediscover this optimal cut.
+#
+# A quantum circuit
+# ~~~~~~~~~~~~~~~~~
+# This section presents the operators used in the QAOA algorithm and their implementation using basic unitary gates.
+# Firstly, if the partition is given by the computational basis states, we can represent the terms in the
+# objective function as operators like so
+#
+# .. math::
+#   C_\alpha = \frac{1}{2}\left(1-\sigma_{z}^j\sigma_{z}^k\right)
+#
+# where the :math:`\alpha\text{th}` edge is between qubits/vertices :math:`(j,k)`.
+# :math:`C_\alpha` is 1 if and only if the :math:`j\text{th}` and :math:`k\text{th}`
+# qubits have different z-axis measurement values, representing separate partitions.
+# The objective function :math:`C` is now a diagonal operator with integer eigenvalues.
+#
+# QAOA starts with a uniform superposition over the :math:`n` bitstring basis states,
+#
+# .. math::
+#   |+_{n}\rangle = \frac{1}{\sqrt{2^n}}\sum_{z=0}^{2^n-1} |z\rangle
+#
+#
+# We aim to explore the space of bitstring states for a superposition which is likely to yield a
+# large value for the :math:`C` operator upon performing a measurement in the computational basis.
+# Using the :math:`2p` angle parameters
+# :math:`\boldsymbol{\gamma} = \gamma_1\gamma_2...\gamma_p`, :math:`\boldsymbol{\beta} = \beta_1\beta_2...\beta_p`
+# we perform a sequence of operations on our initial state:
+#
+# .. math::
+#   |\boldsymbol{\gamma},\boldsymbol{\beta}\rangle = U_{B_p}U_{C_p}U_{B_{p-1}}U_{C_{p-1}}...U_{B_1}U_{C_1}|+_n\rangle
+#
+# where the operators have the explicit forms
+#
+# .. math::
+#   U_{B_i} &= e^{-i\beta_iB} = \prod_{j=1}^n e^{-i\beta_i\sigma_x^j} \\
```

Comment:
```suggestion
#   U_{B_j} &= e^{-i\beta_jB} = \prod_{j=1}^n e^{-i\beta_j\sigma_x^j} \\
```
Shouldn't use `i` both as an index and as sqrt(-1)

Reply:
Thank you